qt/settingsdialog.py

```diff
@@ -561,14 +561,10 @@ def __init__(self, parent):
             .connect(self.excludeCustomSortOrder)
 
         layout.addWidget(self.listExclude)
-        self.listExcludeCount = 0
```

Comment:
This variable is related to the "second invisible column". Couldn't find a code location where it is used.